src/main/java/com/elevenpaths/tacyt/ExternalApiAplicationRequest.java

```diff
@@ -52,6 +52,7 @@
     private String directDownloadURL;
     private String linkToOfficialMarket;
     private Date findDate;
+    private Date deadDate;
```

Comment:
"deadDate" should be in a different class. Add new app doesn't accept deathDate, only update accept it.
Create a new class extending this one and add "deathDate" as new field.

Reply:
Will change definition of US. No necesary